target/linux/imx6/image/lumi.mk

```diff
@@ -0,0 +1,33 @@
+#
+
+define Device/xiaomi_gateway-lumi
+  DEVICE_VENDOR := Xiaomi
+  DEVICE_MODEL := Lumi Gateway
+  BOARD_NAME := xiaomi,gateway-lumi
+  DEVICE_PACKAGES := kmod-bluetooth \
+  	kmod-sound-core kmod-button-hotplug kmod-bluetooth \
+  	bluez-daemon bluez-libs bluez-utils alsa-utils hostapd \
+  	wpa-supplicant ca-certificates kmod-gpio-button-hotplug \
+  	kmod-i2c-core kmod-iio-core nand-utils
+
+  KERNEL_INSTALL := 0
+  KERNEL_NAME := zImage
+  KERNEL_SUFFIX := -zImage
+  KERNEL := kernel-bin
+  IMAGES := sysupgrade.bin dtb rootfs.bin
+  IMAGE/sysupgrade.bin := sysupgrade-tar | append-metadata
+  IMAGE/rootfs.bin := append-rootfs
+  IMAGE/dtb := install-dtb
+
+  DEVICE_DTS := \
+	imx6ull-xiaomi-lumi
+endef
+
+
+define Device/xiaomi_dgnwg05lm
+  $(Device/xiaomi_gateway-lumi)
```

Comment:
Why would we create a common node for a single device?

Reply:
There is a twin device from Aqara  - ZHWG11LM, it has a different rtl8189es wifi chip without integrated Bluetooth, an extra i2c chip for Homekit certificates, and Zigbee chip with an amplifier. Other hardware is identical.
I'd like to add it later but there is no 8189es driver in the kernel right now. So I made a preparation for it.

Reply:
Okay, fine.